### PR TITLE
Work on 2 layers: cafe and restaurant

### PR DESCRIPTION
**Pull request by yjacolin**
* add new symbols (cafe, bar, etc.)
* add new classes for the two layers

(Nothing to change in admin interface)

### CODE REVIEW

File: mapserver/osm.map.mako [hunk 1]
```diff
@@ -359,6 +430,89 @@ LAYER
     END
 END
 
+LAYER
+    NAME "nightlife"
```
Comment (sbrunner) [inline mapserver/osm.map.mako]: Café and Bar => nightlife?
Looks strange for me

**Comment by sbrunner** [comment]
Just a non functional comment, also that's looks good :-)

File: mapserver/osm.map.mako (hunk 1 above)
Comment (yjacolin) [inline mapserver/osm.map.mako]: yes but I can't find a better name :) any idea?

Comment (sbrunner) [inline mapserver/osm.map.mako]: Why not 'snacks'?

Comment (yjacolin) [inline mapserver/osm.map.mako]: I changed the layer name to _entertainment_ and _sustenance_

**Comment by sbrunner** [comment]
OK, merge :-)

**Pull request merged**